packages/fortinet_fortigate/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -285,27 +334,37 @@ processors:
             !ctx._temp.internal_interfaces.contains(ctx.observer.ingress.interface.name)
           )
         )
-  - remove:
-      field:
-        - _temp.time
-        - _temp
-        - syslog5424_sd
-        - fortinet.firewall.tz
-        - fortinet.firewall.date
-        - fortinet.firewall.devid
-        - fortinet.firewall.eventtime
-        - fortinet.firewall.time
-        - fortinet.firewall.duration
+  # Normalize the network direction
+  - script:
+      lang: painless
+      ignore_failure: true
+      params:
+        outgoing: outbound
+        incoming: inbound
+      source: >-
+        def k = ctx.network?.direction.toLowerCase();
+        def normalized = params.get(k);
+        if (normalized != null) {
+            ctx.network.direction = normalized;
+            return
+        }
+        ctx.network.direction = k;
+      if: "ctx.network?.direction != null"
+  # Fix up network direction field to match ECS-allowable values.
+  - set:
+      field: network.direction
+      value: unknown
+      if: "ctx.network?.direction != null && !(['ingress', 'egress', 'inbound', 'outbound', 'internal', 'external'].contains(ctx.network.direction))"
+  - rename:
+      field: fortinet.firewall.interface
+      target_field: observer.ingress.interface.name
       ignore_missing: true
-  - pipeline:
-      name: '{{ IngestPipeline "event" }}'
-      if: "ctx.fortinet?.firewall?.type == 'event'"
-  - pipeline:
-      name: '{{ IngestPipeline "traffic" }}'
-      if: "ctx.fortinet?.firewall?.type == 'traffic'"
-  - pipeline:
-      name: '{{ IngestPipeline "utm" }}'
-      if: "ctx.fortinet?.firewall?.type == 'utm' || ctx.fortinet?.firewall?.type == 'dns'"
+      if: "ctx.observer?.ingress?.interface?.name == null && ctx.network?.direction != null && (['ingress', 'inbound', 'internal'].contains(ctx.network.direction))"
+  - rename:
+      field: fortinet.firewall.interface
+      target_field: observer.egress.interface.name
+      ignore_missing: true
+      if: "ctx.observer?.egress?.interface?.name == null && ctx.network?.direction != null && (['egress', 'outbound', 'external'].contains(ctx.network.direction))"
```

```suggestion
      if: ctx.observer?.egress?.interface?.name == null && ['egress', 'outbound', 'external'].contains(ctx.network.direction)
```
The [] does not contain null, so this is semantically identical. (same above)